framework/utils/randomUtils.py

```diff
@@ -84,107 +86,156 @@ def testSampling(self, n=1e5):
   # this is needed for now since we need to split the stoch enviroments
   distStochEnv = findCrowModule('distribution1D').DistributionContainer.instance()
   boxMullerGen = BoxMullerGenerator()
+  # not sure if this boxMullerGen still needed, will be test this line after first commit
```

Comment:
Did you resolve this comment already, or does it still apply?

Reply:
Still need the boxMullerGen. Removing this line.